sdk/easm/azure-defender-easm/samples/sample1_managing_discovery_runs.py

```diff
@@ -0,0 +1,41 @@
+'''
```

Comment:
Not blocking for this PR - but I think there's some metadata that needs to go at the top of each sample to allow it to be indexed for search in the Azure docs.

Reply:
I've found something that looks right in another folder of the repo and used that as the template